happi/tests/test_cli.py

```diff
@@ -100,6 +100,23 @@ def test_cli_no_argument(runner: CliRunner):
     assert 'Commands:' in result.output
 
 
+def test_repair_name(runner: CliRunner, bad_happi_cfg: str):
+    search_string = '_id=tst_id'
+
+    # make sure the name and id are different
+    with search.make_context('search', [search_string], obj=bad_happi_cfg) as ctx:
+        res = search.invoke(ctx)
+    assert all([r['_id'] != r['name'] for r in res])
```

Comment:
This confused me since there's only one item in the `bad_happi_cfg` where the id and name don't match, but the test is totally still valid so I can't complain too much

Reply:
The search as Laura mentioned in our dev zoom makes it so it picks the invalid item - so `all()` here is really only checking that one